README.md

```diff
@@ -81,11 +81,12 @@ Kubernetes version.
 
 #### Compatibility Matrix
 
-| kops version | k8s 1.5.x | k8s 1.6.x | k8s 1.7.x | k8s 1.8.x |
-|--------------|-----------|-----------|-----------|-----------|
-| 1.8.x        | Y         | Y         | Y         | Y         |
-| 1.7.x        | Y         | Y         | Y         | N         |
-| 1.6.x        | Y         | Y         | N         | N         |
+| kops version       | k8s 1.5.x | k8s 1.6.x | k8s 1.7.x | k8s 1.8.x | k8s 1.9.x |
+|--------------------|-----------|-----------|-----------|-----------|-----------|
+| 1.9.x (unreleased) | Y         | Y         | Y         | Y         | Y         |
+| 1.8.x              | Y         | Y         | Y         | Y         | N         |
```

Comment:
We have typos .. 1.9 should be marked as supporting 1.9 -> 1.6 ... unless I am reading this chart wrong 😆

Reply:
Pretty sure one of us is ;) 

I see:  `1.9.x (unreleased) | Y         | Y         | Y         | Y         | Y        `

Reply:
I cannot read